index.ts

```diff
@@ -71,6 +71,6 @@ export function toMatchJSON(this: MatcherState, received: unknown, jsonObject: a
     return matchers.toMatchObject.call(this, JSON.parse(received as string), jsonObject)
 }
 
-export function jsonContaining(received: unknown, jsonObject: any): ExpectationResult {
-    return toMatchJSON.call(expect.getState() as MatcherState, received, jsonObject)
+export function jsonContaining(this: MatcherState, received: unknown, jsonObject: any): ExpectationResult {
```

Comment:
This change seems safe, but did you try it against 29.3 or older version?

Reply:
I will accept your change. Thanks for looking into it and contributing to the repository.

A lot has changed on the last Jest/Expect versions, so I will take care of upgrading this package to latest. I will release a new version in a couple of hours.